volume-checker/main.go

```diff
@@ -0,0 +1,74 @@
+// Package main checks to make sure the nfs volume has mounted properly
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+)
+
+func main() {
+	http.HandleFunc("/", healthCheck)
+	http.HandleFunc("/read", readDir)
+	http.HandleFunc("/ping", pingAddress)
+
+	if err := http.ListenAndServe(":8080", nil); err != nil {
```

Comment:
This should get the PORT env var and not hard code to 8080 (example: https://github.com/GoogleCloudPlatform/cloud-run-samples/blob/main/gcloud-report/invoke.go#L32-L39)

Reply:
Changed to set port to env var if set